mlflow/lightgbm.py

```diff
@@ -551,6 +561,8 @@ def autolog(
                        If ``False``, trained models are not logged.
                        Input examples and model signatures, which are attributes of MLflow models,
                        are also omitted when ``log_models`` is ``False``.
+    :param log_datasets: If ``True``, datasets are logged as MLflow datasets.
```

Comment:
Which datasets are logged? Training? Validation? Both? 

Can we be more specific here and also update the other autologging integration docstrings for `log_datasets` to be more specific?